Extensions/PrimitiveDrawing/shapepainterruntimeobject.ts

```diff
@@ -337,6 +345,26 @@ namespace gdjs {
       return this._clearBetweenFrames;
     }
 
+    setAntialiasingOn(value: boolean): void {
+      this._antialiasing = value;
+    }
+    
+    isAntialiasingOn(): boolean {
+      return this._antialiasing;
+    }
```

Comment:
```suggestion
    setAntialiasing(enabled: boolean): void {
      this._antialiasing = enabled;
    }
    
    isAntialiased(): boolean {
      return this._antialiasing;
    }
```

For consistency with other function names